### ISSUE TITLE
Language German: 2nd Edition Cards and Names are not available in German

### ISSUE THREAD

**Issue by LordEisenfaust** [sumpfork/dominiontabs#160]
The Card text and the card names are not available in German. Instead, I get the english texts

**Comment by wvoigt** [comment]
Correct.  They need to be translated to German. See the file dominiontabs/domdiv/card_db/de/cards_de.json
Any entry with "untranslated" needs a translation for the part listed.
If you are able, we could use your help.

**Comment by sumpfork** [comment]
#193

**Comment by sumpfork** [comment]
#233 added more

**Comment by JanisBe** [comment]
I just notice this issue, I wrote about polish translation here https://github.com/sumpfork/dominiontabs/issues/263

**Comment by AlessandroFasse** [comment]
Hey Sumpfork. I would like to assist you on this case. I saw that the translations for `Menagerie` are completely missing. That is why I created my own fork and a corresponding branch [here](https://github.com/AlessandroFasse/dominiontabs/tree/german_translations).

I then added in the tests a small line to create the `pdf` file and to check if I am doing it correctly. However, for example the entry for "Black Cat", which I created [here](https://github.com/AlessandroFasse/dominiontabs/blob/german_translations/card_db_src/de/cards_de.json#L2699) is not in the final `pdf`. I think the solution to the problem is easy. Can you give me a little hint? Then I will continue with the rest and also check/verify all other German translations.

**Comment by sumpfork** [comment]
> Hey Sumpfork. I would like to assist you on this case. I saw that the translations for `Menagerie` are completely missing. That is why I created my own fork and a corresponding branch [here](https://github.com/AlessandroFasse/dominiontabs/tree/german_translations).
> 
> I then added in the tests a small line to create the `pdf` file and to check if I am doing it correctly. However, for example the entry for "Black Cat", which I created [here](https://github.com/AlessandroFasse/dominiontabs/blob/german_translations/card_db_src/de/cards_de.json#L2699) is not in the final `pdf`. I think the solution to the problem is easy. Can you give me a little hint? Then I will continue with the rest and also check/verify all other German translations.

It looks to me like you didn't run `doit update_languages` as mentioned [here](https://github.com/sumpfork/dominiontabs#translations). This produces the version of the card database that actually gets packaged with the python code. Wenn ich das mache kriege ich eine "Schwarze Katze" im PDF ohne Probleme.

**Comment by AlessandroFasse** [comment]
Many thanks for the help. I knew it was something simple that I overlooked :) Will continue with the rest and send a Pull Request :)

**Comment by AlessandroFasse** [comment]
Added a pull request: https://github.com/sumpfork/dominiontabs/pull/365

**Comment by poet-of-the-fall** [comment]
Well, I already got all the texts for the German cards in my project but unfortunately in latex...
But if it helps just copy whatever you need. My project is here: https://github.com/poet-of-the-fall/carddivider
